src/fluent-plugin-mdsd/lib/fluent/plugin/out_mdsd.rb

```diff
@@ -181,6 +182,15 @@ def get_schema_value_str(record)
         return resultStr
     end
 
+    # Convert (unify) mdsd.syslog.** tags to mdsd.syslog so that mdsd sees only
+    # one single tag for all syslog messages. This is the use case for basic
+    # syslog messages collection. Also, it appears that tags can't be changed
+    # by a fluentd filter, so they need to be changed here in this output plugin.
+    def create_mdsd_source(tag)
+        if tag.start_with?("mdsd.syslog.")
```

Comment:
>"mdsd.syslog. [](start = 27, length = 13)

This design will require that fluentd configuration file to use 'mdsd.syslog' as the syslog tag. This is not ideal because the purpose of using tag is that a user can use any tag in fluentd config. one alternative is to make this as a out_mdsd plugin parameter (see line 21). then update the related sample and documents.